src/components/Header.jsx

```diff
@@ -4,9 +4,6 @@ import covid from '../img/covid.svg';
 const Header = () => (
   <nav className="navbar bg-blue f-gill fc-white">
     <div className="container-fluid d-flex justify-content-around">
-      <Link to="/">
-        <box-icon name="left-arrow" animation="burst-hover" color="#ffffff"></box-icon>
-      </Link>
```

Comment:
This is part of the project requirement, Kindly put this back.
As the previous code reviewer stated, It should not link to the home page but the previous page.

Reply:
Thanks for the feedback, now I understand, I used useNavigate to jump between pages. 😃